swarm/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# 0.40.1 - [unreleased]
```

Comment:
`Cargo.toml` bump on `libp2p-swarm` is missing.

Reply:
right, thanks for reminding. There is a version mismatch between the released version of `libp2p-swarm` and it's entry on the `CHANGELOG.MD`. Bumped to `0.40.2` and created https://github.com/libp2p/rust-libp2p/pull/3063 to address it, probably should be merged first.